portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,7 +49,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery preparedQuery = datastore.prepare(query);
 
     // Set maximum number of comments to be included in the response.
-    int numOfComments = Integer.parseInt(request.getParameter("comment-limit"));
+    String numOfCommentsString = request.getParameter("comment-limit");
```

Comment:
This looks good! In the above comment, can you note that you fallback to 0 for invalid input?

Reply:
Done!